syntax = "proto3";

package SDPackage;

// HGW Authentication & Key Exchange Phase Service


// { PID MU ,M3 ,C2 ,VMUG }
message AuthKES3Req{
  string PID = 1;
  string M3 = 2;
  string C2 = 3;
  string V = 4;
}
// { M4 ,VSD }
message AuthKES3Res{
  string M4 = 1;
  string V = 2;
}

service SDAuthKE{
  rpc AuthKES3(AuthKES3Req) returns (AuthKES3Res) {}
}

